src/components/header/Header.tsx

```diff
@@ -18,17 +18,17 @@ export const Header = () => {
     <Styled.Header>
       <Styled.Container>
         <Styled.Wrapper>
-          <Link href="/">
-            <a>team awesome</a>
-          </Link>
-          <button onClick={handleClick}>{isActive ? <Close /> : <Hamburger />}</button>
+          <Link href="/">team awesome</Link>
+          <button onClick={handleClick} type="button">
+            {isActive ? <Close /> : <Hamburger />}
```

Comment:
It's easier to read if you add an indicator that these components are icons:)

Reply:
What do you mean with indicator?

Reply:
I would suffix icon components with the word `icon` so it becomes `CloseIcon` and `HamburgerIcon`. It makes them easier to read